spec/integ/crypto/crypto.spec.ts

```diff
@@ -1242,7 +1243,7 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
                                     content: { algorithm: "m.megolm.v1.aes-sha2" },
                                 }),
                                 testUtils.mkMembership({
-                                    mship: "join",
+                                    mship: KnownMembership.Join,
```

Comment:
for the record, I am very ambivalent about using symbolic constants in test code. Using the literal value gives an opportunity to check that the symbolic constant is working as you think it is, and can make the test easier to understand.

(At one point we had a bug in Synapse, where  the unit tests were misusing what we *thought* was a symbolic constant, but which turned out not to be, and it ended up masking a real bug.)

But I don't have a strong feeling. Happy to stick with it now it's done.

Reply:
Interesting, I thought I was the only person who thought that way.

I think we should leave it as I've done it, since it matches the approach we use in this codebase, but yeah in general I agree.